journals/apps/journals/models.py

```diff
@@ -144,7 +140,7 @@ def get_discovery_data(self):
             'card_image_url': self.journal_about_page.card_image_absolute_url,
             'short_description': self.journal_about_page.short_description,
             'full_description': self.journal_about_page.long_description,
-            'status': 'active' if self.journal_about_page.live else 'inactive',
+            'status': 'active' if self.publish else 'inactive',
```

Comment:
what is the reason for changing this line?

Reply:
before when new journal was publishing, status for other services (i.e: discovery and e-commerce) were decided on base of journal_about_page, regardless of what `publish` param is.
I just make it sure status on other services should be according to publish param given in publish_journal command.